apps/src/code-studio/initApp/project.js

```diff
@@ -542,21 +542,14 @@ var projects = module.exports = {
     if (typeof arguments[0] === 'function' || !sourceAndHtml) {
       // If no source is provided, shift the arguments and ask for the source
       // ourselves.
-      var args = Array.prototype.slice.apply(arguments);
+      const args = Array.prototype.slice.apply(arguments);
```

Comment:
I always forget about this, and am pro-es6ifying everything as we go.  Thanks for pointing this out.